### PR TITLE
Doc: Update hld-trace-log.rst

### PR DESCRIPTION
**Pull request by KaigeFu**
1) removed the earlylog section as we have remove
     the earlylog support from acrn hypervisor.
  2) updated SOS ACRN Log Module section as the size
     and base address of acrn log buffer are configured
     by kernel option now instead of hardcoding.

Signed-off-by: Kaige Fu <kaige.fu@intel.com>

### CODE REVIEW

File: doc/developer-guides/hld/hld-trace-log.rst [hunk 1]
```diff
@@ -182,24 +182,14 @@ of a single log message is 320 bytes. Log messages with a length between
...
-0x6DE00000 is reserved for acrn log.  This space is further divided into
-two each ranges, one for current run and one for last previous run:
-
-.. figure:: images/log-image59.png
-   :align: center
-
-   ACRN Log crash log/current log buffers
+ACRNLog module provides one kernel option `hvlog=$size@$pbase` to configure
+the size and base address of acrn log buffer. This space will be further divided
+into two ranges, last log buffer and current log buffer.
```
Comment (lyan3) [inline doc/developer-guides/hld/hld-trace-log.rst]: two buffers with equal size: last log buffer and current log buffer.

File: doc/developer-guides/hld/hld-trace-log.rst [hunk 2]
```diff
@@ -182,24 +182,14 @@ of a single log message is 320 bytes. Log messages with a length between
...
-To enable retrieving log messages from a crash, 4MB of memory from
-0x6DE00000 is reserved for acrn log.  This space is further divided into
-two each ranges, one for current run and one for last previous run:
-
-.. figure:: images/log-image59.png
-   :align: center
-
-   ACRN Log crash log/current log buffers
+ACRNLog module provides one kernel option `hvlog=$size@$pbase` to configure
+the size and base address of acrn log buffer. This space will be further divided
```
Comment (lyan3) [inline doc/developer-guides/hld/hld-trace-log.rst]: acrn log buffer -> hypervisor log buffer

File: doc/developer-guides/hld/hld-trace-log.rst (hunk 1 above)
Comment (KaigeFu) [reply doc/developer-guides/hld/hld-trace-log.rst]: Fixed.

File: doc/developer-guides/hld/hld-trace-log.rst (hunk 2 above)
Comment (KaigeFu) [reply doc/developer-guides/hld/hld-trace-log.rst]: Fixed.

**Pull request merged**